### ISSUE TITLE
Vncinstall

### ISSUE THREAD

**Issue by ilserver** [maxnet/berryboot#579]
Hello, I try ti use the vncinstall feature but if I don't specify the ip address in cmdline.txt I can't find the ip provide by the dhcp server to berryboot.

I am looking for a feature that print to screen the ip where berryboot can be managed by vnc client so I can use use it without do a network scan to get the right ip.

Thanks,

**Comment by radimkohout** [comment]
The IP is written on the bottom of menu page, if you are connected over wifi or ethernet like: [xxx.xxx.xxx.xxx]

**Comment by ilserver** [comment]
No, the ip is visible only if vncinstall option is not specify, where berryboot GUI start before image start. If the option vncinstall is use (no ip specify so it can get the ip from dhcp), i get only a black text screen with 2 line on top that tell me I can use vnc (but no ip where it answer).
So : is there a way to print the ip where vnc server run when I use vncinstall option in the cmdline.txt file?
